ui/src/pages/session-details/playback/activity-plot/activity-plot.tsx

```diff
@@ -69,14 +90,30 @@ export const ActivityPlot = ({
               zeroline: false,
               rangemode: 'nonnegative',
               fixedrange: true,
+              range: [yAxisMin, yAxisMax],
+              side: 'left',
             },
-            xaxis: {
-              showline: true,
+            yaxis2: {
               showgrid: false,
               showticklabels: false,
               zeroline: false,
+              rangemode: 'nonnegative',
+              fixedrange: true,
+              range: [0, session.detectionsMaxCount],
```

Comment:
For example (regarding comment above):

```suggestion
              range: [0, session.detectionsMaxCount ?? yAxisMax],
```